### PR TITLE
WIP: API docs

### PR DESCRIPTION
**Pull request by austinwbest**
#### Database Migration
 NO

#### Description
Updates to API documentation. Using the PR system for validation purposes.

### CODE REVIEW

File: src/Radarr.Api.V3/swagger.json [hunk 1]
```diff
@@ -2,15 +2,12 @@
...
-      "url": "https://radarr.video"
-    },
-    "version": "3.0.0"
+    "version": "3.0.0",
+    "description": "The way users should interact with Radarr programatically. To utilize any of these endpoints you will need a few pieces of information:\n\n* hostPath: localhost, 10.1.0.1, 192.168.1.1, etc\n* hostPort: 7878 (unless you modify it)\n* apiKey: Located in Settings > General > Security"
   },
   "servers": [
     {
-      "url": "{protocol}://{hostPath}/api/v3",
+      "url": "{protocol}://{hostPath}:{hostPort}/api/v3",
```
Comment (Qstick) [inline src/Radarr.Api.V3/swagger.json:10]: `hostPort` is not needed, its part of `hostPath` in the default below... most will change it to a revproxied url to test which will not have a port. If testing with a port just include it in the `hostPath` like the example has it

File: src/Radarr.Api.V3/swagger.json [hunk 2]
```diff
@@ -2,15 +2,12 @@
   "openapi": "3.0.0",
   "info": {
     "title": "Radarr",
-    "description": "Movie Automation",
-    "contact": {
-      "url": "https://radarr.video"
-    },
-    "version": "3.0.0"
+    "version": "3.0.0",
+    "description": "The way users should interact with Radarr programatically. To utilize any of these endpoints you will need a few pieces of information:\n\n* hostPath: localhost, 10.1.0.1, 192.168.1.1, etc\n* hostPort: 7878 (unless you modify it)\n* apiKey: Located in Settings > General > Security"
```
Comment (Qstick) [inline src/Radarr.Api.V3/swagger.json:6]: remove reference to hostPort

File: src/Radarr.Api.V3/swagger.json [hunk 3]
```diff
@@ -216,16 +251,517 @@
...
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Invalid API Key"
+          }
+        },
+        "operationId": "get-diskspace",
+        "description": "Get diskspace information\n\nLocation: System > Status",
```
Comment (Qstick) [inline src/Radarr.Api.V3/swagger.json:557]: What's the location for?

**Comment by austinwbest** [comment]
Closing this & pushing it to radarr branch instead

**Pull request closed without merging**